pom.xml

```diff
@@ -8,7 +8,7 @@
     <artifactId>autotune</artifactId>
     <version>0.2</version>
     <properties>
-        <fabric8-version>4.13.2</fabric8-version>
+        <fabric8-version>6.13.4</fabric8-version>
```

Comment:
Looks like `7.0.0` was released 3 days ago ... can we check if that works for us

Reply:
Sure ...

Reply:
@dinogun I have updated the version to `7.0.0` and tested the PoC code and the current code. In version 7.0.0, the structure has changed, and the `VerticalPodAutoscaler` has been moved to the `autoscaling` API group. I will keep this change in mind and update the VPA code accordingly in upcoming PRs. It should not cause any issues.